### ISSUE TITLE
How to provide scalatest options in gradle-scalatest

### ISSUE THREAD

**Issue by h-ngo** [maiflai/gradle-scalatest#70]
We want to experiment with some flag options to configure scalatest reporter (http://www.scalatest.org/user_guide/using_the_runner, "Configuring reporters").

How do we provide these options to gradle-scalatest?

**Comment by maiflai** [comment]
The plugin tries to respect the gradle test logging configuration by manipulating the reporters, so this is not really something that is publicly exposed.

What sort of configuration are you thinking of?

**Comment by maiflai** [comment]
That's odd - https://github.com/maiflai/gradle-scalatest/issues/66 suggests that it should work.

What does the rest of the build file look like? Which version of the plugin are you using?

**Comment by h-ngo** [comment]
Ah. I think I know why that works. It works for test task, but our tests are trigger by testScoverage. How do we apply testLogging to testScoverage?

**Comment by maiflai** [comment]
You might try applying this to all test tasks?

```groovy
tasks.withType(Test) {
  testLogging { 
    events = ['FAILED'] 
  }
}
```

**Comment by h-ngo** [comment]
Yeah. that's exactly what we applied. our test setup is a bit complicated cuz we used scalatest to test both Java and Scala code. that blocks only work for tests against Java, not Scala. Perhaps, the setting is not correct. Will try again and update here.

We are using gradle-scalatest 0.16

**Comment by maiflai** [comment]
Ok - I don't think there's anything in later releases of the plugin that would specifically affect this.

The block above should work for both; although I'm not certain about the details of testScoverage when handling a Java-only project.

If you could share a simple reproduction project that would help enormously.
